source/api/discovery/0.5/index.md

```diff
@@ -56,15 +56,15 @@ This specification leverages existing techniques, specifications, and tools in o
+The objective of the IIIF Change Discovery API is to provide a machine to machine API that provides the information needed to discover and subsequently make use of IIIF resources. The intended audience is other IIIF-aware systems that can leverage the content and APIs. While this work may benefit others outside of the IIIF community directly or indirectly, the objective of the API is to specify an interoperable solution that best and most easily fulfills the discovery needs within the community of participating organizations.
 
 The discovery of IIIF resources requires a consistent and well understood pattern for content providers to publish lists of links to their available content. This allows a baseline implementation of discovery systems that process the list, looking for resources that have been added or changed.
 
-This process can be optimized by allowing the content providers to publish descriptions of when their content has changed, enabling consuming systems to only retrieve the resources that have been modified since they were last retrieved. These changes might include when content is deleted or otherwise becomes unavailable. Finally, for rapid synchronization, a system of notifications pushed from the publisher to a set of subscribers can reduce the amount of effort required to constantly poll all of the systems to see if anything has changed.
+This process can be optimized by allowing the content providers to publish descriptions of how their content has changed, enabling consuming systems to only retrieve the resources that have been modified since they were last retrieved. These changes might include when content is deleted or otherwise becomes unavailable. Finally, for rapid synchronization, a system of notifications pushed from the publisher to a set of subscribers can reduce the amount of effort required to constantly poll multiple sources to see if anything has changed.
 
-Work that is out of scope of this API includes the recommendation or creation of any descriptive metadata formats, and the recommendation or creation of metadata search APIs or protocols. The diverse domains represented within the IIIF community already have successful standards fulfilling these use cases, and the results of previous attempts to reconcile these standards across domains have not seen widespread adoption. Also out of scope is optimization of the transmission of content, for example recommendations about transferring any source media or data between systems.
+Work that is out of scope of this API includes the recommendation or creation of any descriptive metadata formats, and the recommendation or creation of metadata search APIs or protocols. The diverse domains represented within the IIIF community already have successful standards fulfilling these use cases. Also out of scope is optimization of the transmission of content, for example recommendations about transferring any source media or data between systems.
```

Comment:
:-1: This is from the discussions that led to the Discovery for Humans CG. It's there to cut short any proposal that a reconciliation of existing standards is not out of scope.

Reply:
> and the results of previous attempts to reconcile these standards across domains have not seen widespread adoption

...is what was cut. I think the point is still there and that clause is a little too didactic.

Reply:
It still says very clearly that inventing new descriptive metadata specs is out of scope

Reply:
:+1: I think it still makes the point clearly

Reply:
👍  I agree, makes the point.